CMakeLists.txt

```diff
@@ -387,7 +387,7 @@ set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
 FetchContent_Populate(${extension_name}
   SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
   GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/slicersalt/${extension_name}
-  GIT_TAG        50ae545559a9a897de60832e29a8062364cae7cf # slicersalt-2018-11-29-4baa99c10
+  GIT_TAG        727cb3ccbe340d01c289b55cbe0f45e27dd64497 # slicersalt-2020-06-11-727cb3c
```

Comment:
In the branch `slicersalt-YYYY-MM-DD-SHA{7}`, `YYYY-MM-DD-SHA{7}` should correspond to the date and sha of the commit common to the upstream project and our fork.

Looking at https://github.com/NIRALUser/GROUPS , we can see that the last commit is 6cd4da0 .and it is from 2018. This means that the branch name should reflect that.

We should also look into integrating changes into the upstream repo

Reply:
Ah. In that case, I made this mistake in all the branches involved.

It seems that to solve this, I should merge the changes into the existing branches, and in the GROUPS repo revert the change in [this commit](https://github.com/slicersalt/GROUPS/commit/4b2d73548b94905d129d5f93b65139dec4e9c2f8). Then all the incorrect branches listed above can be removed.

Does this seem correct?

Reply:
I will summarize below the changes to be done. 

If something is unclear, I am available for a hangout anytime.

### RigidAlignement

To understand what I described below, running `gitk --all` will gives a nice overview:

![image](https://user-images.githubusercontent.com/219043/84917778-c29a7080-b08d-11ea-9993-8dc923077fa5.png)

In this case, branch `slicersalt-2018-11-16-ca0c6dfff1` should be updated to point to the last commit you published.

```
git clone git://github.com/slicersalt/RigidAlignment
cd RigidAlignment
git checkout -b slicersalt-2018-11-16-ca0c6dfff1  9386fdb
git push origin slicersalt-2018-11-16-ca0c6dfff1
```

Then delete branch `slicersalt-2020-06-09-9386fdb`:

```
git push origin :slicersalt-2020-06-09-9386fdb
``` 



### GROUPS

To understand what I described below, running `gitk --all` will gives a nice overview:

![image](https://user-images.githubusercontent.com/219043/84912968-fffbff80-b087-11ea-859c-cc42d8cfd585.png)

We can observe few issues:
* the change https://github.com/slicersalt/GROUPS/commit/7b99735500ee10e8ed7a5c670f85a1fb7439417f  (ENH: Updates for Python 3 and new SPV) is not part of the latest branch anymore
* there are unneeded branches

Following the steps below will address these issues:

* Step 1: Checkout a branch called `slicersalt-2018-11-16-15c897e` and cherry-pick your changes:

```
git clone git@github.com:slicersalt/GROUPS
cd GROUPS
git checkout -b slicersalt-2018-11-16-15c897e 7b99735   # Branch is based of commit "ENH: Updates for Python 3 and new SPV"
```

* Step 2: Then cherry-pick your changes:

```
git cherry-pick 7368826..4b2d735
```

**Note:** Since there are conflicts integrating your first change, I assumed your file already included all relevant changes from https://github.com/slicersalt/GROUPS/commit/7b99735500ee10e8ed7a5c670f85a1fb7439417f. You may want to double check it is the cas


Running `gitk` should confirm everything is okay.

* Step 3: Finally, publish the updated branch:

```
git push origin slicersalt-2018-11-16-15c897e
```


* Step 4: Finally, these branches should be removed (branches have been incorrectly named prior to your changes):
```
git push origin :slicersalt-2019-10-03-15c897e
git push origin :slicersalt-2020-05-26-15c897e
git push origin :slicersalt-2020-06-09-4b2d735
```

Reply:
Thank you so much for your help! That was very informative. 

I've done all the steps above, and also on the SPHARM-PDM repository.

```
$ cd SPHARM-PDM/

$ git checkout -b slicersalt-2018-11-29-4baa99c10 727cb3c
$ git push origin slicersalt-2018-11-29-4baa99c10 

$ git push -d origin slicersalt-2020-06-09-60d892b 
$ git push -d origin slicersalt-2020-06-11-727cb3c 
```

I also updated the GIT_TAG comments in the CMakeLists for the Groups and SlicerSALT repositories to use the correct branch names. The only tag that changed was the one in SlicerSALT which now refers to the change in GROUPS. The commits are here:

- [GROUPS@437c4ff](https://github.com/slicersalt/GROUPS/commit/437c4ff6aea17c2139130ca1c7247ce8250f8863)
- [SlicerSALT@71834e8](https://github.com/allemangD/SlicerSALT/commit/71834e874d097f63a9059fccd26054881dd6cfb2)

Thanks so much for the clarification!